docs/how-to/spacetimedb-standalone.md

```diff
@@ -0,0 +1,250 @@
+# Self Hosting SpacetimeDB
+
+This tutorial will guide you through setting up SpacetimeDB on an Ubuntu 24.04 server, securing it with HTTPS using Nginx and Let's Encrypt, and configuring a systemd service to keep it running.
+
+## Prerequisites
+- A fresh Ubuntu 24.04 server (VM or cloud instance of your choice)
+- A domain name (e.g., `example.com`)
+- `sudo` privileges on the server
+
+Also, join the community! We have an official discord server: https://discord.gg/spacetimedb
+
+## Step 1: Create a Dedicated User for SpacetimeDB
+For security purposes, create a dedicated `spacetimedb` user to run SpacetimeDB with a home directory at `/stdb`:
+
+```sh
+sudo mkdir /stdb
+sudo useradd --system spacetimedb
+sudo chown -R spacetimedb:spacetimedb /stdb
+```
+
+Switch to the new user and install SpacetimeDB:
+
+```sh
+sudo -u spacetimedb bash -c 'curl -sSf https://install.spacetimedb.com | sh -s -- --root-dir /stdb --yes'
+```
+
+## Step 2: Create a Systemd Service for SpacetimeDB
+To ensure SpacetimeDB runs on startup, create a systemd service file:
+
+```sh
+sudo nano /etc/systemd/system/spacetimedb.service
+```
+
+Add the following content:
+
+```ini
+[Unit]
+Description=SpacetimeDB Server
+After=network.target
+
+[Service]
+ExecStart=/stdb/spacetime --root-dir=/stdb start --listen-addr='127.0.0.1:3000'
+Restart=always
+User=spacetimedb
+WorkingDirectory=/stdb
+
+[Install]
+WantedBy=multi-user.target
+```
+
+Enable and start the service:
+
+```sh
+sudo systemctl daemon-reload
+sudo systemctl enable spacetimedb
+sudo systemctl start spacetimedb
+```
+
+Check the status:
+
+```sh
+sudo systemctl status spacetimedb
+```
+
+## Step 3: Install and Configure Nginx
+
+### Install Nginx
+
+```sh
+sudo apt update
+sudo apt install nginx -y
+```
+
+### Configure Nginx Reverse Proxy
+Create a new Nginx configuration file:
+
+```sh
+sudo nano /etc/nginx/sites-available/spacetimedb
+```
+
+Add the following configuration, remember to change `example.com` to your own domain:
+
+```nginx
+server {
+    listen 80;
+    server_name example.com;
+
+    location / {
+        proxy_pass http://localhost:3000;
+        proxy_http_version 1.1;
+        proxy_set_header Upgrade $http_upgrade;
+        proxy_set_header Connection "Upgrade";
+        proxy_set_header Host $host;
+    }
+
+    # This restricts who can publish new databases to your SpacetimeDB instance. We recommend
+    # restricting this ability to local connections. You should uncomment this before you use a
+    # SpacetimeDB server in production.
+    # location /v1/publish {
+        # allow 127.0.0.1;
+        # deny all;
+        # proxy_pass http://localhost:3000;
+        # proxy_http_version 1.1;
+        # proxy_set_header Upgrade $http_upgrade;
+        # proxy_set_header Connection "Upgrade";
+        # proxy_set_header Host $host;
+    # }
```

Comment:
I would have this uncommented, and have the comment suggest that the reader might want to comment this out if they have controlled access to the server some other way. This tutorial is about deploying SpacetimeDB-standalone for production, so it should show the production setup by default.

Reply:
I originally had it uncommented and then I chickened out 😆 I agree with your analysis, I'll change it 👍